sdk/keyvault/Azure.Security.KeyVault.Certificates/samples/Sample1_HelloWorld.cs

```diff
@@ -41,7 +41,7 @@ public void HelloWorldSync()
             {
                 certOp.UpdateStatus();
 
-                Thread.Sleep(certOp.PollingInterval);
+                Thread.Sleep(TimeSpan.FromSeconds(1));
```

Comment:
cc @heaths, samples should not point customers in direction of manual polling.

Reply:
Noted, though I didn't write the original code. I'll be sure to avoid it / watch out for it in the future.